server/player.lua

```diff
@@ -135,16 +137,15 @@ exports('SetJob', SetJob)
 function SetJobDuty(identifier, onDuty)
     local player = type(identifier) == 'string' and (GetPlayerByCitizenId(identifier) or GetOfflinePlayer(identifier)) or GetPlayer(identifier)
 
-    if not player then return end
-
-    player.PlayerData.job.onduty = not not onDuty
-
-    if player.Offline then return end
-
-    TriggerEvent('QBCore:Server:SetDuty', player.PlayerData.source, player.PlayerData.job.onduty)
-    TriggerClientEvent('QBCore:Client:SetDuty', player.PlayerData.source, player.PlayerData.job.onduty)
+    if not player then
+        error(('SetJobDuty couldn\'t find player with identifier %s'):format(identifier))
```

Comment:
```suggestion
        lib.print.error(('SetJobDuty couldn\'t find player with identifier %s'):format(identifier))
```
This is a breaking change as `error` will cause the parent code to stop running without a `pcall`. Let's change it to `lib.print.error` which doesn't error out.

Reply:
You're right, I forgot error stops execution